src/utils/generator.js

```diff
@@ -113,6 +125,16 @@ export class GeneratorEntry {
   getDependencies(): void | Array<Generator> {
     invariant(false, "GeneratorEntry is an abstract base class");
   }
+
+  notEqualToAndDoesNotHappenAfter(entry: GeneratorEntry): boolean {
```

Comment:
I think the names are flipped now. This one should be called `....DoesNotHappenBefore`, and the other one `...DoesNotHappenAfter`.